Tests/Tactics/ReduceFetchInst.lean

```diff
@@ -0,0 +1,13 @@
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author(s): Alex Keizer
+-/
+import Tactics.Reflect.FetchAndDecode
+import Tests.«AES-GCM».GCMGmultV8Program
+
+/-! ## Tests for `refuceFetchInst` simproc -/
+
+example (h : s.program = GCMGmultV8Program.gcm_gmult_v8_program) :
```

Comment:
Let's also add a test for a longer program, say SHA512. Is all well or is maxRecDepth reached when the last instruction is fetched?

Reply:
I've added tests for the first and last instructions of SHA512. Both are instantaneous (after the simplification described in the next comment)